prepare/cards/ethos.py

```diff
@@ -35,6 +35,16 @@
                 output_format="{label}",
                 postprocessors=["processors.take_first_non_empty_line"],
             ),
+            InputOutputTemplate(
+                input_format="Given this sentence: {text}. Classify if it contains hatespeech. Choices: {choices}. I would classify this sentence as: ",
```

Comment:
It's "hate speech" more comon than "hatespeech"? Or is this the expected output?

Reply:
These were the original choices and therefore this is the expected output. It works this way. I can try "hatespeech" instead and check. should work as well

Reply:
The choices are "hate speech" and "not hate speech" so why are the instructions "Classify if it contains hatespeech"? (without a space).  Maybe it does not make a difference, but it seems wrong.